app/controllers/pages_controller.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 class PagesController < ApplicationController
-  def home
-  end
 
   def get_highlights
     @highlights = Highlight.where(promoted: true).take(4)
   end
+
+  def index
```

Comment:
Your route points to the `:ambler` method, not the index method.

Reply:
Not sure where index came from or why it needed all buildings. I may have copied and pasted when setting up the controller. I've now removed the index method as we won't ever need one.